tracks/csharp/exercises/sum-of-multiples/README.md

```diff
@@ -0,0 +1,60 @@
+### Reasonable solutions
+
+```csharp
+using System.Linq;
+using System.Collections.Generic;
+
+public static class SumOfMultiples
+{
+    public static int Sum(IEnumerable<int> inputs, int max)
+        => inputs.SelectMany(
+            input => Enumerable.Range(1, (max - 1) / input).Select(numTimes => numTimes * input)
```

Comment:
Could you rename `input` to `multiple`?

Reply:
The example implementation is a bit shorter than this version:

```csharp
public static int Sum(IEnumerable<int> multiples, int max)
{
    return Enumerable.Range(1, max - 1)
        .Where(i => multiples.Any(m => i % m == 0))
        .Sum();
}
```

To me, that version is a bit more readable. What do you think of including that version?

Reply:
done